### PR TITLE
#0: Add functional tests for 2CQ ResNet-50

### PR DESCRIPTION
**Pull request by tt-asaigal**
### Ticket
Link to Github Issue

### Problem description
Add tests to verify 2 CQ Resnet-50 PCC.

### What's changed
Describe the approach used to solve the problem.
Summarize the changes made and its impact.

### Checklist
- [ ] Post commit CI passes
- [ ] Blackhole Post commit (if applicable)
- [x] Model regression CI testing passes (if applicable)
- [ ] Device performance regression CI testing passes (if applicable)
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

**Review by mywoodstock — APPROVED** [review]

File: .github/workflows/tg-frequent-tests-impl.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
         run: tar -xvf ttm_${{ matrix.test-group.arch }}.tar
       - uses: ./.github/actions/install-python-deps
       - name: Run frequent regression tests
-        timeout-minutes: 60
+        timeout-minutes: 90
```
Comment (TT-billteng) [inline .github/workflows/tg-frequent-tests-impl.yaml:39]: are we hitting the timeout limit?

Comment (tt-asaigal) [reply .github/workflows/tg-frequent-tests-impl.yaml:39]: Yeah we are, with the 2 additional resnet tests :(

**Comment by tt-asaigal** [comment]
Passing TG Frequent with the timeout increased: https://github.com/tenstorrent/tt-metal/actions/runs/11225798225
Takes ~80 minutes

File: .github/workflows/tg-frequent-tests-impl.yaml (hunk 1 above)
Comment (tt-aho) [reply .github/workflows/tg-frequent-tests-impl.yaml:39]: We should start splitting this into separate jobs by model/category similar to t3k freq now that we have more tests (not in this pr).

Comment (ttmchiou) [reply .github/workflows/tg-frequent-tests-impl.yaml:39]: please file a ticket for splitting into separate jobs

**Review by ttmchiou — APPROVED** [review]

File: .github/workflows/tg-frequent-tests-impl.yaml (hunk 1 above)
Comment (tt-asaigal) [reply .github/workflows/tg-frequent-tests-impl.yaml:39]: Work for this is tracked here: https://github.com/tenstorrent/tt-metal/issues/13610

**Pull request merged**